deps/neverbleed/neverbleed.c

```diff
@@ -1346,20 +1346,51 @@ static void *daemon_conn_thread(void *_sock_fd)
     return NULL;
 }
 
+
+static void neverbleed_cleanup_fds(int listen_fd, int close_notify_fd)
```

Comment:
I think we can remove the prefix ("neverbleed_") and call the function "cleanup_fds" here> It is a static function, we do not need to be concerned of name collision.

Reply:
Done